test/unit/question_base_test.rb

```diff
@@ -15,8 +15,8 @@ class QuestionBaseTest < ActiveSupport::TestCase
 
     should 'raise exception if next_node is invoked multiple times' do
       e = assert_raises do
-        @question.next_node :outcome_one
-        @question.next_node :outcome_two
+        @question.next_node { outcome :one }
+        @question.next_node { outcome :two }
       end
```

Comment:
Minor. I've add a comment to the Trello card about moving away from outcome names prefixed with "outcome", and why having it can be useful.